lib/commonmark.js

```diff
@@ -86,7 +86,7 @@ function renderHtml(markdownString, options, cb) {
 }
 
 function parse(markdownString, cb) {
-  return cb(null, commonmarkRenderer.parse(markdownString));
+  return cb(null, commonmarkRenderer.parse(markdownString, {}));
```

Comment:
please first do a try/catch with retrieving the results of the `.parse` method; to be able to call callback with an Error, or with `null, results`